spinners.json

```diff
@@ -1181,5 +1181,165 @@
 			"βββββρβ",
 			"ββββββρ"
 		]
+	},
+	"fingerDance": {
+		"interval": 160,
+		"frames": [
+			"☝️",
+			"✌️",
+			"🤘",
+			"🤟",
+			"🖖",
+			"🖐",
+			"✋",
+			"🤚",
+			"👆"
+		]
+	},
+	"fistBump": {
+		"interval": 80,
+		"frames": [
+			"🤜    🤛",
+			"🤜    🤛",
+			"🤜    🤛",
+			" 🤜  🤛",
+			"  🤜🤛",
+			" 🤜✨🤛",
+			"🤜 ✨ 🤛"
+		]
+	},
+	"soccerHeader": {
+		"interval": 80,
+		"frames": [
+			" 🧑⚽️       🧑",
+			"🧑  ⚽️      🧑",
+			"🧑   ⚽️     🧑",
+			"🧑    ⚽️    🧑",
+			"🧑     ⚽️   🧑",
+			"🧑      ⚽️  🧑",
+			"🧑       ⚽️🧑 ",
+			"🧑      ⚽️  🧑",
+			"🧑     ⚽️   🧑",
+			"🧑    ⚽️    🧑",
+			"🧑   ⚽️     🧑",
+			"🧑  ⚽️      🧑"
+		]
+	},
+	"mindblown": {
+		"interval": 160,
+		"frames": [
+			"😐",
+			"😐",
+			"😮",
+			"😮",
+			"😦",
+			"😦",
+			"😧",
+			"😧",
+			"🤯",
+			"💥",
+			"✨",
+			"",
+			"",
+			""
+		]
+	},
```

Comment:
I'm also not sure about that one tbh. Yeah, it's "just" emoji and "just" a cli-spinner, but it looks pretty graphic and insensible. I think the same about the already implemented "grenade" spinner btw.

Reply:
I like this one. It's fun.

Reply:
Okay then :)